### DISCUSSION TITLE
Download QRZ logs to wavelog

### DISCUSSION THREAD

**Discussion by Bleschke** [wavelog/wavelog#600] [Ideas]
Feature request: Download all logs from QRZ using API key (with and without confirmations... all logs made). 

I use other programs to log contacts (ex. N3FJP). The current version of Wavelog does not download logs, only uploads them.  Downloading logs from QRZ would serve as a backup and an alternative method to logging contacts with Wavelog.

**Comment by HB9HIL** [comment]
Hello @Bleschke 

We discussed that in team and we agreed that this is not something which Wavelog was made for. It's a first level log software and we trying our very best to improve the functionality for this every day. For this reason we support uploading QSOs and downloading confirmations from QRZ.com. 

Unfortunately we do not see a "Download QSO's from QRZ.com" Feature in Wavelog as it is build to do it the other way around. Log QSO's in Wavelog and Upload to QRZ.com. I'm sorry that I can't give you a better answer here.

My suggestion: 

Use N3FJP and pipe all QSO's to Wavelog. From here you can use the third-party services all in one place, like LotW, QRZ.com, eQSL and so on. I don't know N3FJP but you may can check out if it has some kind of UDP ADIF Broadcast to work with [WavelogGate](https://github.com/wavelog/WaveLogGate). 

Thank you for using Wavelog.

Moved this to discussions (ideas)

**Comment by MikleBas** [comment]
I would like to raise a question - how can I download QSOs from QRZ.COM?
I have already registered all the API keys from QRZ.COM, checked the login and password 10 times - and even on the new log (Wavelog)
I get a response from QRZ.COM as in the screenshot
**`Downloaded QRZ report contains no matches.`**
What am I doing wrong?
P.S. I understand that you need a paid subscription to download the log there, but you can download it for free, right?
I would be grateful for an answer
73!
